pages/getting-started/quick-start/generate.mdx

```diff
@@ -70,203 +52,188 @@ The diagram above represents the relationships of your database.
 
 ```ts snaplet.config.ts
-/// <reference path=".snaplet/snaplet.d.ts" />
-import { defineConfig } from 'snaplet';
-import { copycat } from '@snaplet/copycat';
-
-export default defineConfig({
-  generate: {
-    async run(snaplet) {
-      await snaplet.posts([{
-        // focus
-        title: 'Hello World!',
-      }]);
+import { SnapletClient } from '@snaplet/client';
+
+const snaplet = new SnapletClient();
+
+await snaplet.posts([
+  {
+    title: 'Hello World!',
+    // focus(1:6)
+    author: {
+      email: ({ seed }) =>
+        copycat.email(seed, {
+          domain: 'acme.org',
+        }),
     },
   },
-});
+]);
 ```
 
 ---
 
-You can also define the values of your model's relationships.
-Here we define the author of the post represented by the `author` field.
+Next, we also define the comments of the post represented by the `comments` field.
+As we need more than one comment, we can use the `x` function which will create a list of similar comments.
+You can read it as "comments times 3".
 
 ```ts snaplet.config.ts
-/// <reference path=".snaplet/snaplet.d.ts" />
-import { defineConfig } from 'snaplet';
-import { copycat } from '@snaplet/copycat';
-
-export default defineConfig({
-  generate: {
-    async run(snaplet) {
-      await snaplet.posts([{
-        title: 'Hello World!',
-        // focus(1:6)
-        author: {
-          email: ({ seed }) =>
-            copycat.email(seed, {
-              domain: 'acme.org'
-            }),
-        },
-      }]);
+import { SnapletClient } from '@snaplet/client';
+
+const snaplet = new SnapletClient();
+
+await snaplet.posts([
+  {
+    title: 'Hello World!',
+    author: {
+      email: ({ seed }) =>
+        copycat.email(seed, {
+          domain: 'acme.org',
+        }),
     },
+    // focus
+    comments: (x) => x(3),
   },
-});
+]);
 ```
 
 ---
 
-Finally, we also define the comments of the post represented by the `comments` field.
-As we need more than one comment, we can use the `x` function which will create a list of similar comments.
-You can read it as "comments times 3".
+Finally, we'll add a `dryRun` option when instantiating `SnapletClient`. When we run our script with a `DRY` environment variable set, we'll log the sql statements
+to the console instead altering the database.
 
 ```ts snaplet.config.ts
-/// <reference path=".snaplet/snaplet.d.ts" />
-import { defineConfig } from 'snaplet';
-import { copycat } from '@snaplet/copycat';
-
-export default defineConfig({
-  generate: {
-    async run(snaplet) {
-      await snaplet.posts([{
-        title: 'Hello World!',
-        author: {
-          email: ({ seed }) =>
-            copycat.email(seed, {
-              domain: 'acme.org'
-            }),
-        },
-        // focus
-        comments: (x) => x(3),
-      }]);
+import { SnapletClient } from '@snaplet/client';
+
+const snaplet = new SnapletClient({
+  // focus
+  dryRun: Boolean(+process.env.DRY)
```

Comment:
@jgoux I needed to do this to fit it in with the `--sql` step we do later in the guide. I added this as the last step in the flow to not overwhelm the reader with too much in the beginning (better they get the core concept of the library first, rather than peripheral things like this, is what I was going for).

Sound ok?

The other thought was to look for a `DRY` env var under the hood for snaplet client so that it does not need to be given explicitly. Its a bit implicit though. Not sure.